From 7dea971f3fa80f1c512eb7033ef4b08c5275e215 Mon Sep 17 00:00:00 2001
From: Wig Cheng <wig.cheng@technexion.com>
Date: Tue, 23 Nov 2021 18:38:19 +0800
Subject: [PATCH 17/18] arm64: dts: add tevi series to imx8mp-evk support list

Include tevi-ov5640, tevi-ar0521, tevi-ar0144 and tevi-ar0234 camera
sensors.
---
 arch/arm64/boot/dts/freescale/Makefile | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 3939c918f8e4..d9178ba7eadf 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -101,7 +101,11 @@ dtb-$(CONFIG_ARCH_MXC) += imx8mp-evk.dtb imx8mp-evk-rm67191.dtb imx8mp-evk-it626
 			  imx8mp-evk-dual-ov2775.dtb imx8mp-evk-spdif-lb.dtb \
 			  imx8mp-evk-sof-wm8960.dtb \
 			  imx8mp-evk-iqaudio-dacplus.dtb imx8mp-evk-iqaudio-dacpro.dtb imx8mp-evk-hifiberry-dacplus.dtb \
-			  imx8mp-evk-usdhc1-m2.dtb imx8mp-evk-rm67199.dtb
+			  imx8mp-evk-usdhc1-m2.dtb imx8mp-evk-rm67199.dtb \
+			  imx8mp-evk-tevi-ov5640-csi1.dtb imx8mp-evk-tevi-ov5640-csi2.dtb \
+			  imx8mp-evk-tevi-ar0521-csi1.dtb imx8mp-evk-tevi-ar0521-csi2.dtb \
+			  imx8mp-evk-tevi-ar0144-csi1.dtb imx8mp-evk-tevi-ar0144-csi2.dtb \
+			  imx8mp-evk-tevi-ar0234-csi1.dtb imx8mp-evk-tevi-ar0234-csi2.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mq-evk.dtb imx8mq-evk-rpmsg.dtb imx8mp-ab2.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mp-ddr4-evk.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mp-evk-ndm.dtb
-- 
2.25.1

